index.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <title>irb</title>
   <link href="css/screen.css" media="screen" rel="stylesheet" type="text/css" />
-  <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
-  <script src="http://d1n0x3qji82z53.cloudfront.net/src-min-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+  <script src="https://d1n0x3qji82z53.cloudfront.net/src-min-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
This would work for connections over http and https, but not anyone browsing locally with file:///. Loading the script over https will work for all three.